lib/origin_simulator/default_recipe.ex

```diff
@@ -0,0 +1,67 @@
+defmodule OriginSimulator.DefaultRecipe do
```

Comment:
Wondering if this could have all just lived in https://github.com/bbc/origin_simulator/blob/master/lib/origin_simulator/recipe.ex and be called by `OriginSimulator.Recipe.default` this could then open up the discussion of whether there could variations of predefined recipes living in the codebase with some kind of toggle.
Happy for this to go as is though just a thought

Reply:
thought at this. Just tried to keep the html part out of the Recipe module